lib/taurus/core/tango/tangoattribute.py

```diff
@@ -448,8 +449,17 @@ def poll(self, **kwargs):
             if single:
                 self.read(cache=False)
             else:
-                self.__attr_value = self.decode(kwargs.get('value'))
+                value = self.decode(kwargs.get('value'))
                 self.__attr_err = kwargs.get('error')
+                filter_old_event = getattr(tauruscustomsettings,
+                                           'FILTER_OLD_TANGO_EVENTS', False)
+                if self.__attr_value is not None and \
+                        self.__attr_err is None and \
+                        filter_old_event and \
+                        kwargs.get('time') < self.__attr_value.time.totime():
```

Comment:
I see a potential problem in this implementation (i do not know if in practice it is important):
`kwargs.get('time') < self.__attr_value.time.totime()` will return True if kwargs does not have the `time` key.  
I think that in this case one should **not** discard the event. Maybe it should give an exception (using `kwargs['time']`), or it should return False (using `kwargs.get('time', float('inf'))`)